include/mbedtls/des.h

```diff
@@ -162,85 +172,106 @@ int mbedtls_des_key_check_key_parity( const unsigned char key[MBEDTLS_DES_KEY_SI
 int mbedtls_des_key_check_weak( const unsigned char key[MBEDTLS_DES_KEY_SIZE] );
 
 /**
- * \brief          DES key schedule (56-bit, encryption)
+ * \brief          Perform a DES key schedule (56-bit, encryption).
  *
- * \param ctx      DES context to be initialized
- * \param key      8-byte secret key
+ * \param ctx      The DES context to use. Must be initialized.
+ * \param key      The secret key.
+ *                 Must be a readable buffer of length 8 Bytes.
```

Comment:
Maybe the constant `MBEDTLS_DES_KEY_SIZE` should be referenced here?

Reply:
I prefer to spell it out, as otherwise there's another indirection for the user.